source/MaterialXView/Editor.cpp

```diff
@@ -95,7 +95,12 @@ void PropertyEditor::addItemToForm(const mx::UIPropertyItem& item, const std::st
 
     if (!group.empty())
     {
-        form.addGroup(group);
+        ng::Label* groupLabel =  new ng::Label(container, group);
+        groupLabel->setFontSize(20);
+        groupLabel->setFont("sans-bold");
+        new ng::Label(container, "");
```

Comment:
This is not an error. It's a 2 cell wide layout so this is to add padding. There is no "separator" or "pad" widget.